DRAFT.md

```diff
@@ -1459,31 +1451,24 @@ constexpr polymorphic(allocator_arg_t, const Allocator& a,
 constexpr polymorphic(polymorphic&& other) noexcept;
 ```
 
-13. _Mandates_: If `allocator_traits<allocator_type>::is_always_equal::value`
-    is `false`, then `T` is a complete type.
-
-14. _Effects_: If `other` is valueless, `*this` is valueless. Otherwise, if
-    `alloc == other.alloc` is `true`, either constructs an object of type
-    `polymorphic` that owns the owned object of other, making `other` valueless;
-    or, owns an object of the same type constructed from the owned object of
-    `other` considering that owned object as an rvalue. Otherwise, if `alloc !=
-    other.alloc` is `true`, constructs an object of type `polymorphic`,
-    considering the owned object in `other` as an rvalue, using the allocator
-    `alloc`.
+9. _Effects_: `alloc` is direct-non-list-initialized with
+  `std::move(other.alloc)`. If `other` is valueless, `*this` is valueless.
+  Otherwise, either `*this` takes ownership of the owned object of `other` or,
+  owns an object of the same type constructed from the owned object of
+  `other` considering that owned object as an rvalue, using the allocator
+  `alloc`.
 
   _[Drafting note: The above is intended to permit a small-buffer-optimization
   and handle the case where allocators compare equal but we do not want to swap
   pointers.]_
 
 ```c++
 constexpr polymorphic(allocator_arg_t, const Allocator& a,
-                      polymorphic&& other) noexcept(allocator_traits<Allocator>::is_always_equal::value);
+                      polymorphic&& other)
+  noexcept(allocator_traits<Allocator>::is_always_equal::value);
 ```
 
-15. _Mandates_: If `allocator_traits<allocator_type>::is_always_equal::value`
-    is `false`, then `T` is a complete type.
-
-16. _Effects_: `alloc` is direct-non-list-initialized with `a`. If `other` is
+10. _Effects_: `alloc` is direct-non-list-initialized with `a`. If `other` is
     valueless, `*this` is valueless. Otherwise, if `alloc == other.alloc` is
     `true`, either constructs an object of type `polymorphic` that owns the
     owned object of other, making `other` valueless; or, owns an object of the
```

Comment:
"that owns the owned object of other" -> "that takes ownership of the owned object of other"